db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,34 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
```

Comment:
OK I understand that gap in data types. Since this one is meant to be small
(configuration) I think clear serialization is more important then efficiency.
So I change my mind :+1: for `:text`